libfreshclam/libfreshclam.c

```diff
@@ -795,7 +803,16 @@ fc_error_t fc_update_databases(
             logg(LOGG_INFO, "    CDN and your own network.\n");
             logg(LOGG_INFO, " 4. Please do not open a ticket asking for an exemption from the rate limit,\n");
             logg(LOGG_INFO, "    it will not be granted.\n");
+            if (0 != g_lastRay[0]) {
```

Comment:
In testing I found that it does not show the `cf-ray` ID the first time are rate limited. E.g., delete your "freshclam.dat" file after you're rate limited and try again. 

A copy of this is needed in two more places:
1. Line 732 
2. Line 972

Reply:
I left it out of those two error messages on purpose because there is no question about what the problem is.  The user is rate limited, and we ask not to put in a ticket.  I was thinking we would only print them if we are suggesting the user put in a ticket.